### PR TITLE
feat v4: rename packages

### LINKED ISSUE #811: 💡 Proposal - v4 package renaming
## The problem

Forma 36 package names are very long and with the new build setup there'll be a lot of typing!

## The proposed solution

Rename existing packages to use the shorter "f36" moniker:

- `@contentful/forma-36-react-components` → `@contentful/f36-components`
- `@contentful/forma-36-tokens` → `@contentful/f36-tokens`

Use the moniker for new packages as well:

- `@contentful/f36-base`
- `@contentful/f36-typography`
- `@contentful/f36-workbench`
- `@contentful/f36-modal`
- `@contentful/f36-icon`
- `@contentful/f36-card`
- `@contentful/f36-button`
- `@contentful/f36-skeleton`
- `@contentful/f36-input`
- `@contentful/f36-form`
- `@contentful/f36-tab`
- `@contentful/f36-note`
- `@contentful/f36-notification`
- `@contentful/f36-switch`
- `@contentful/f36-spinner`
- `@contentful/f36-tag`
- `@contentful/f36-dropdown`
- `@contentful/f36-accordion`
- `@contentful/f36-autocomplete`

## Later work

Maybe we can group some packages in a core package:

`@contentful/f36-inputs`, `@contentful/f36-forms`, `@contentful/f36-base`, and `@contentful/f36-typography` → `@contentful/f36-core`

We should cut this from the initial scope since it makes sense to first investigate what packages our users would want grouped into such a core package. See https://github.com/contentful/forma-36/issues/811#issuecomment-768113163

### PR DESCRIPTION
**Pull request by denkristoffer**
# Purpose of PR

This renames packages as planned in #811 

`@contentful/forma-36-react-components` → `@contentful/f36-components`
`@contentful/forma-36-tokens` → `@contentful/f36-tokens`
`@contentful/forma-36-website` → `@contentful/f36-website`

`@contentful/forma-36-fcss` has not been renamed. I'd rather delete it 🤭 

Closes #811

# Follow up required

We need to deal with Datepicker and Timepicker somehow.

### CODE REVIEW

**Review by suevalov — CHANGES_REQUESTED** [review]
Can we keep folder names untouched and only rename package names for the time being? 

It would be easier for merging changes from v3 and will create less noise in V4 pull request.

**Comment by denkristoffer** [comment]
@suevalov Fair point, thanks. Fixed.

**Review by suevalov — COMMENTED** [review]
I'd be also in favor of bumping all packages manually to version 4.0.0 in package.json

**Comment by denkristoffer** [comment]
Yeah I was wondering about that, does semantic release just continue on from the set version?

**Comment by suevalov** [comment]
@denkristoffer yes, the semantic release should pick it up from package.json

**Review by suevalov — APPROVED** [review]
👍🏼 from me, but let's collect a couple of more reviews before merging this one

**Review by gui-santos — APPROVED** [review]
giphy

File: packages/forma-36-fcss/package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@contentful/forma-36-fcss",
   "description": "Forma 36 FCSS",
-  "version": "0.3.1",
+  "version": "4.0.0",
```
Comment (mshaaban0) [inline packages/forma-36-fcss/package.json:4]: 🔥

**Review by mshaaban0 — APPROVED** [review]

**Pull request merged**